docs/guides/auto-backup/workload/index.md

```diff
@@ -37,556 +39,713 @@ namespace/demo created
 
->Stash keeps all backup data encrypted. So, snapshot files in the bucket will not contain any meaningful data until they are decrypted.
+If we check the YAML of the `Snapshot`, we can find the information about the backed up components of the StatefulSet.
 
-## Backup StatefulSet
+```bash
+$ kubectl get snapshots -n demo sts-repo-statefulset-sample-sts-frequent-backup-1704431400 -oyaml
+```
 
-Now, we are going to backup a StatefulSet with the same blueprint we have used to backup Deployment in the previous section.
+```yaml
+apiVersion: storage.kubestash.com/v1alpha1
+kind: Snapshot
+metadata:
+  labels:
+    kubestash.com/app-ref-kind: StatefulSet
+    kubestash.com/app-ref-name: sample-sts
+    kubestash.com/app-ref-namespace: demo
+    kubestash.com/repo-name: sts-repo
+  name: sts-repo-statefulset-sample-sts-frequent-backup-1704431400
+  namespace: demo
+spec:
+  ...
+status:
+  components:
+    dump-pod-0:
+      driver: Restic
+      duration: 6.232547841s
+      integrity: true
+      path: repository/v1/frequent-backup/pod-0
+      phase: Succeeded
+      resticStats:
+      - hostPath: /source/data
+        id: 07abd0bd6d28406141709d01b343dba35fea3198321cd60dbe1710088c239153
+        size: 26 B
+        uploaded: 1.397 KiB
+      size: 848 B
+    dump-pod-1:
+      driver: Restic
+      duration: 6.384220849s
+      integrity: true
+      path: repository/v1/frequent-backup/pod-1
+      phase: Succeeded
+      resticStats:
+      - hostPath: /source/data
+        id: 06d34f5fa7a2251408a7d846fb0616d5156a387106fe344854ab4a095c79bfa3
+        size: 26 B
+        uploaded: 1.397 KiB
+      size: 849 B
+    dump-pod-2:
+      driver: Restic
+      duration: 6.18345431s
+      integrity: true
+      path: repository/v1/frequent-backup/pod-2
+      phase: Succeeded
+      resticStats:
+      - hostPath: /source/data
+        id: caa69ec25c20fcfe26624ecd30032054b0b68789ada42f2c8eb8554e60d5fc3a
+        size: 26 B
+        uploaded: 1.395 KiB
+      size: 850 B
+ ...
+```
+> For StatefulSet, KubeStash takes backup from every pod of the StatefulSet. Since we are using three replicas, three components are present in the Snapshot. KubeStash uses `dump-pod-<ordinal-value>` as the component name for StatefulSet. The ordinal value in the component's name represents the ordinal value of the StatefulSet pod ordinal.
+
+Now, if we navigate to `demo/sample-sts/repository/v1/frequent-backup/` directory of our GCS bucket, we are going to see that the components of the StatefulSet have been stored there and each of the components contains encrypted data.
+
```

Comment:
```suggestion
> For StatefulSet, KubeStash takes backup from every pod of the StatefulSet. Since we are using three replicas, three components have been backed up. KubeStash uses `dump-pod-<ordinal-value>` as the component name where <ordinal-value> corresponds to the pod's ordinal number within the StatefulSet.
```

> This change applies to other places

Reply:
Also incorporate the relevant reviews from the workload backup